extensions/kinesis/src/test/java/com/hazelcast/jet/kinesis/KinesisIntegrationTest.java

```diff
@@ -87,6 +87,8 @@ public KinesisIntegrationTest() {
     @BeforeClass
     public static void beforeClass() {
         assumeDockerEnabled();
+        //Newer version of localstack with arm64 support fails in KinesisIntegrationTest
+        assumeNoArm64Architecture();
```

Comment:
Do you have some issue or details what fails exactly?

Reply:
Created https://github.com/hazelcast/hazelcast/issues/23692